purchase_request_to_procurement/models/purchase_request.py

```diff
@@ -73,6 +73,7 @@ def _prepare_procurement_order_values(
             'product_qty': self.product_qty,
             'product_uom': self.product_uom_id.id,
             'date_planned': self.date_required,
+            'request_id': self.request_id.id,
```

Comment:
Just an idea, but maybe we could link the purchase request line directly to the procurement. What do you think @lreficent ? If we have a purchase request which contains two lines with the same product, we can't know what is the true origin of the procurement orders.

And I think that ```request_id``` field is declared in ```purchase_request_procurement``` so we can't use it here.

Reply:
Finally I moved the field definition to purchase_request.